lib/visual_metrics.py

```diff
@@ -331,9 +353,12 @@ def BuildGvizDataTable(datatable, metric, baseline_codec, other_codecs):
       # codec's metric
       if (this_codec in datatable and filename in datatable[this_codec]
           and filename in datatable[baseline_codec]):
+        this_dataset = ExtractBitrateAndPsnr(datatable,
+                                             this_codec,
+                                             filename,
+                                             full_results)
         overall = DataSetBetter(
```

Comment:
It computes how much one data set is better than the other. See function comment.

Not changing the name in this CL.